helm/fullnode/values.yaml

```diff
@@ -15,7 +15,9 @@ rust_log: info
 
 image:
   repo: diem/validator
-  tag: release-1.1_a1abfc34
+  # "devnet" tag is updated daily
+  # additional image tags can be found on DockerHub: https://hub.docker.com/r/diem/validator/tags
```

Comment:
Update the /tools images below too?

Reply:
PFN is actually failing to connect to testnet with the new tag:

```
2021-03-29T21:10:29.636686Z [state-sync] ERROR state-sync/src/coordinator.rs:220 {"error":"NoAvailablePeers(\"No peers to send chunk request to!\")","event":"fail","name":"progress_check"}
2021-03-29T21:10:29.736391Z [state-sync] WARN state-sync/src/coordinator.rs:1512 {"name":"timeout","version":0}
2021-03-29T21:10:29.736438Z [state-sync] WARN state-sync/src/coordinator.rs:1565 {"event":"missing_peers","name":"send_chunk_request"}
2021-03-29T21:10:29.736783Z [state-sync] ERROR state-sync/src/coordinator.rs:1545 {"error":"NoAvailablePeers(\"No peers to send chunk request to!\")","event":"send_chunk_request_fail","local_epoch":1,"name":"timeout","version":0}
2021-03-29T21:10:29.737118Z [state-sync] ERROR state-sync/src/coordinator.rs:220 {"error":"NoAvailablePeers(\"No peers to send chunk request to!\")","event":"fail","name":"progress_check"}
```

Reply:
repro'ed locally, and reverting back to the old release tag `release-1.1_a1abfc34` resolves the crash loop caused by state sync

Reply:
cc @gregnazario ? We can also just use the `testnet` tag, but it doesn't seem to be up to date with the rev currently running in testnet

Reply:
probably related to https://github.com/diem/diem/pull/7966

Reply:
But that PR hasn't landed yet? Maybe I'm missing something

Reply:
Got it! Updated the config with the new layout, and marked the seed role as "Upstream"